src/Window.cpp

```diff
@@ -1003,7 +1077,9 @@ void Window::dispatchCommand(QString command) {
             clearWall(x, y, direction);
         }
         else {
-            ASSERT_NEVER_RUNS();
+            stopRun("Stop running because of an exceptional command!");
+            //ASSERT_NEVER_RUNS();
```

Comment:
This is a concerning change. Previously, this branch was never hit. Why is it being hit now?